src/passes/Print.cpp

```diff
@@ -684,6 +684,15 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     lastPrintedLocation = { 0, 0, 0 };
     printOpening(o, "func ", true);
     printName(curr->name);
+    if (currModule && !minify) {
+      // emit the function index in a comment
+      for (Index i = 0; i < currModule->functions.size(); i++) {
```

Comment:
Do we really not keep this anywhere in a better-indexed format, so that we have to do this loop on every function? Could we do it just once (on init, or visitModule or something)?

Reply:
It is `O(N^2)`, yeah. I can add a utility to do this.

Reply:
Added a commit with an indexing utility used by both binary emitting and this comment code.